src/ophyd_async/core/device_save_loader.py

```diff
@@ -177,8 +177,22 @@ def set_signal_values(
     for phase_number, phase in enumerate(values):
         # Key is signal name
         for key, value in phase.items():
+            # Skip ignored values
+            if value is None:
```

Comment:
Loading from a yaml file where we had a list of ignored PVs wouldn't work without this

Reply:
We have a 'Compression' enum now, in `NDFileHDF`: https://github.com/bluesky/ophyd-async/blob/main/src/ophyd_async/epics/areadetector/writers/nd_file_hdf.py

Any devices using this, get serialised (at the moment) so that if the compression is set to "None", it appears in the yaml as:

```
some-signal: None
```
obviously this is bad yaml; it should be 'null' or something similar. However, for the moment, this change would ignore that pv completely, which seems against what we want to happen.

But this should probably be a broader, post i22 discussion.

Reply:
Don't we want to completely ignore these values?

Reply:
No, this is not actually bad yaml, just confusing yaml. `some-signal: None` translates to python `{"some-signal": "None"}`, which is what we actually need, while `some-signal: null` translates to python `{"some-signal": None}` which would the the output if we wanted to ignore that field.

Either way I think what you have here is correct, but we need to document this confusing corner case